frontend/public/components/software-details.jsx

```diff
@@ -110,76 +97,24 @@ export const SoftwareDetails = connectToFlags(FLAGS.OPENSHIFT)(
         .catch(() => this.setState({kubernetesVersion: 'unknown'}));
     }
 
-    _checkTectonicVersion() {
-      coFetchJSON('api/tectonic/version')
-        .then((data) => {
-          this.setState({
-            tectonicVersion: data.version,
-            tectonicVersionObj: data
-          });
-        })
-        .catch(() => this.setState({tectonicVersion: 'unknown'}));
-    }
-
-    _checkAppVersions() {
-      k8sGet(AppVersionModel).then((appversions) => {
-        const tectonicTPR = _.find(appversions.items, (a) => a.metadata.name === clusterAppVersionName);
-        if (tectonicTPR) {
-          this.setState({currentTectonicVersion: tectonicTPR.status.currentVersion});
-        }
-      }).catch(() => this.setState({currentTectonicVersion: 'unknown'}));
-    }
-
     _checkOpenshiftVersion() {
       coFetchJSON('api/kubernetes/version/openshift')
         .then((data) => {
           this.setState({openshiftVersion: data.gitVersion, openshiftVersionObj: data});
         }).catch(() => this.setState({openshiftVersion: 'unknown'}));
     }
 
-    _checkCloudProvider() {
-      k8sGet(ConfigMapModel, 'tectonic-config', 'tectonic-system').then((configMap) => {
-        this.setState({cloudProviders: [_.get(configMap, 'data.installerPlatform', null)]});
-      }, () => this.setState({cloudProviders: ['UNKNOWN']}));
-    }
-
     render() {
-      const {openshiftVersion, currentTectonicVersion, tectonicVersion, kubernetesVersion, cloudProviders} = this.state;
+      const {openshiftVersion, kubernetesVersion} = this.state;
       const openshiftFlag = this.props.flags[FLAGS.OPENSHIFT];
 
       if (flagPending(openshiftFlag)) {
         return null;
       }
 
-      return <div>
-        <SoftwareDetailRow
-          title="Kubernetes"
-          detail={kubernetesVersion}
-          text="Kubernetes version could not be determined."
-        />
-        {openshiftFlag ? (
-          <SoftwareDetailRow
-            title="OpenShift"
-            detail={openshiftVersion}
-            text="OpenShift version could not be determined."
-          />
-        ) : (
-          <React.Fragment>
-            <SoftwareDetailRow
-              title="Tectonic"
-              detail={currentTectonicVersion || tectonicVersion}
-              text="Tectonic version could not be determined."
-            />
-            {cloudProviders &&
-            <SoftwareDetailRow
-              title="Cloud Provider"
-              detail={cloudProviderNames(cloudProviders)}
-              text="Cloud Provider could not be determined."
-            />
-            }
-          </React.Fragment>
-        )}
-      </div>;
+      return <React.Fragment>
+        <SoftwareDetailRow title="Kubernetes" detail={kubernetesVersion} text="Kubernetes version could not be determined." />
```

Comment:
Kind of strange "text" is really error text.  Nothing to fix here, just a comment.